### PR TITLE
Feature/issue 98/change password api

### PR DESCRIPTION
**Pull request by Buicongbang04**
#98

### CODE REVIEW

**Review by nnh53 — COMMENTED** [review]
remove it

**Review by nnh53 — COMMENTED** [review]
bang said not use so delete it

**Review by nnh53 — CHANGES_REQUESTED** [review]
test request change

**Review by nnh53 — DISMISSED** [review]
@Buicongbang04

**Review by nnh53 — COMMENTED** [review]
fix

File: src/index.ts [hunk 1]
```diff
@@ -19,6 +19,19 @@ const whitelist = [
     'http://localhost:3000',
     'https://nikeclonetraining-be-project.onrender.com/'
 ]
+
```
Comment (nnh53) [inline src/index.ts]: delete this

File: src/modules/user/user.services.ts [hunk 2]
```diff
@@ -76,6 +77,13 @@ class UsersService {
         const user = await databaseService.users.findOne({ password })
         return Boolean(user)
     }
+    async finduser(user_id: string, password: string) {
+        const user = await databaseService.users.findOne({
+            _id: new ObjectId(user_id),
```
Comment (nnh53) [inline src/modules/user/user.services.ts:82]: @Buicongbang04 said not use

**Comment by Buicongbang04** [comment]
Process from develop to main later

**Pull request closed without merging**